README.md

```diff
@@ -19,6 +19,8 @@ Supports `table`, `discovery`, `coordination`, `ratelimiter`, `scheme`, `scripti
 
 ## Installation
 
+`ydb-go-sdk` supports all Go versions supported by the official Go Release Policy. That is, the two most recent releases of Go.
```

Comment:
Suggest to add link to the policy:

## Supported Go Versions

`ydb-go-sdk` supports all Go versions supported by the official
[Go Release Policy](https://go.dev/doc/devel/release#policy). That is,
the two most recent releases of Go.